tests/integration/src/test/scala/mesosphere/marathon/integration/facades/MesosFacade.scala

```diff
@@ -150,4 +151,31 @@ class MesosFacade(val url: String, val waitTime: FiniteDuration = 30.seconds)(im
       Done
     }
   }
+
+  /**
+    * Drain agent using v1 operator API
+    *
+    * @return Right(Done) on success, Left(errorString) otherwise
```

Comment:
I'm not really sure what you mean with `Right` and `Left`. The return type is `RestResult`.

Reply:
I copied/pasted the signature and doc from `markAgentGone` – thanks for catching this; the docs there are also wrong.